### DISCUSSION TITLE
Python Code Samples

### DISCUSSION THREAD

**Discussion by intelliot** [XRPLBounties/Proposals#32] [Closed (claimed or cancelled bounty)]
### Discussion thread for bounty: [Python Code Samples](https://github.com/XRPLBounties/Proposals/blob/main/bounties/0032%20Python%20Code%20Samples.md) ([PR](https://github.com/XRPLBounties/Proposals/pull/8))

**Comment by jonathanlei** [comment]
Milestone 6 by @wojake approved and merged here: https://github.com/XRPLF/xrpl-dev-portal/commit/f5ff555be8da9b2eec5e320eaf922f23f62554a1

**Reply by jonathanlei** [reply]
Milestone 1 submission by @kasiimh1 here: https://github.com/XRPLF/xrpl-dev-portal/pull/1501/files

**Reply by jonathanlei** [reply]
Milestone 2 submission by @ObiajuluM here: https://github.com/XRPLF/xrpl-dev-portal/pull/1492

**Reply by jonathanlei** [reply]
Milestone 3 submission by @ObiajuluM  here: https://github.com/XRPLF/xrpl-dev-portal/pull/1504

**Reply by jonathanlei** [reply]
Milestone 4 submission by @ripci504 @ approved: https://github.com/XRPLF/xrpl-dev-portal/pull/1499

**Reply by jonathanlei** [reply]
Milestone 5 submissions approved: 
by @YossiBenAri here: https://github.com/XRPLF/xrpl-dev-portal/pull/1509
by @wojake here: https://github.com/XRPLF/xrpl-dev-portal/pull/1493

**Comment by jonathanlei** [comment]
Milestone 7 submission by @wojake here: https://github.com/XRPLF/xrpl-dev-portal/pull/1502

**Comment by jonathanlei** [comment]
Milestone 8 submission by @ObiajuluM here: https://github.com/XRPLF/xrpl-dev-portal/pull/1497

**Comment by jonathanlei** [comment]
Milestone 1 submission approved & merged by @wojake here: https://github.com/XRPLF/xrpl-dev-portal/commit/951106567095e7afe243c76e32c8f1346a75ee2c

**Comment by JST5000** [comment]
We have accepted the following milestones as completed and are working to pay out each contributor in full. 

Milestone 1: @wojake and @Kasiimh1
Milestone 4: @ripci504
Milestone 5: @wojake and @YossiBenAri
Milestone 6: @wojake
Milestone 7: @wojake

In some cases multiple contributors produced unique high quality code around the same time so we decided to award multiple contributors for the bounty. This is decided on a case-by-case basis.

**Comment by ckniffen** [comment]
We’ve accepted @ObiajuluM's submission for milestone 8 for this bounty. 

To confirm your GitHub account, @ObiajuluM please reply to this message with the confirmation number (It will be provided via a follow up email).

**Reply by ObiajuluM** [reply]
hello @ckniffen here is the confirmation number you requested **5485**

**Comment by JST5000** [comment]
We’ve accepted @ObiajuluM’s submission for milestone 2 for this bounty.
